projects/go-lib/src/lib/components/go-datepicker/calendar-views.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../../styles/variables';
+
+.go-calendar__header {
```

Comment:
Some of these BEM selectors don't fully adhere to best practices, but I don't think they should hold up the PR. We can always circle back and update them later.